interface/patient_file/summary/demographics.php

```diff
@@ -1853,6 +1853,11 @@ function setMyPatient() {
                     $("#eligibility").get(0).scrollIntoView();
                 }
             });
+            <?php
+                $GLOBALS["kernel"]
+                    ->getEventDispatcher()
+                    ->dispatch(RenderEvent::EVENT_RENDER_JAVA, new RenderEvent($pid), 10);
```

Comment:
Method signature has changed.
Should be `dispatch(new RenderEvent($pid), RenderEvent::EVENT_RENDER_JAVA, 10)`

Soon we'll have to refactor all our events as current argument placement is deprecated.
"Calling the "Symfony\Component\EventDispatcher\ " method with the event name as first argument is deprecated since Symfony 4.3, pass it second and provide the event object first instead."